app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+                >
+              </li>
+            </ul>`;
+    } else if (i === 4 || i === 7) {
+      desktopProjectDiv.classList.add('project-3', 'project-block');
+      desktopProjectDiv.innerHTML = `<h3 class="project-title">${recentProjects[4].name}</h3>
+            <p class="project-details">
+              ${recentProjects[4].description}
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  ${recentProjects[4].workStack[0]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[4].workStack[1]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[4].workStack[2]}</a
+                >
+              </li>
+            </ul>`;
+    } else if (i === 5) {
+      desktopProjectDiv.classList.add('project-3', 'project-block');
+      desktopProjectDiv.innerHTML = `<h3 class="project-title">${recentProjects[5].name}</h3>
+            <p class="project-details">
+             ${recentProjects[5].description}
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  ${recentProjects[5].workStack[0]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  >${recentProjects[5].workStack[1]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  >${recentProjects[5].workStack[2]}</a
+                >
+              </li>
+            </ul>
+          `;
+    }
+
+    if (desktopProjectDiv.innerHTML !== '') {
+      descktopProjectsWrapperDiv.appendChild(desktopProjectDiv);
+    }
+  }
+}
+
+// portfolio end
+
+// popup start
+
+const modal = document.querySelector('.project-details-popup');
+const overlay = document.querySelector('.overlay');
+const openModalBtn = document.querySelectorAll('.pop');
+
+overlay.classList.add('hidden');
+const popupContainer = document.getElementById('project-details-popup');
```

Comment:
- [ ] As per the design, the div containing the popup modal should have a blur effect to hide the elements behind it. This effect adds a nice touch of visual appeal and enhances the user experience. I kindly recommend you to implement this blur effect as specified in the design to achieve the desired look and feel for your project. Great job so far, and keep up the good work! 🚀🎨👍

![image](https://github.com/FilbertKimasa/Portfolio/assets/55840999/5ede47a6-9323-497e-b237-07797e990042)


- [ ] I also noticed that in the desktop version, the popup is currently positioned at the top of the screen, whereas the design specifies it should be almost in the middle of the screen. To align with the design requirements, I recommend updating your code to position the popup at the center of the screen on the desktop view.

|  design: | yours |
| --- | ---- |
| ![image](https://github.com/FilbertKimasa/Portfolio/assets/55840999/eb4cb188-ea26-427c-9298-8f9b44b8c2cb) | ![image](https://github.com/FilbertKimasa/Portfolio/assets/55840999/c581638b-edec-477f-bd5f-65d4cf57e453) |